source/client-side-encryption/etc/update-corpus.py

```diff
@@ -0,0 +1,28 @@
+import json
+import sys
+
+description = """Update the existing corpus data from a file with only deterministically encrypted fields.
```

Comment:
I added this script to make it easier to update the corpus data without updating the random fields to make a simpler diff.